presto-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/singlenode-kerberos-hdfs-impersonation-with-data-protection/iceberg.properties

```diff
@@ -0,0 +1,15 @@
+connector.name=iceberg
```

Comment:
I guess iceberg catalog is not used in the test.

Reply:
Dropped